[fmiweather] Cannot add discovered things from inbox in OH3 / cannot create things manually

OK## Expected Behavior

- Adding FMI weather things from inbox works without errors
- creating things manually would be possible

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->

When trying to add discovered FMI weather thing, it fails with an error in the logs:

```
23:56:11.518 [INFO ] [ig.discovery.internal.PersistentInbox] - Added new thing 'fmiweather:observation:station_100996_Helsinki_Harmaja' to inbox.
23:56:11.518 [INFO ] [smarthome.event.InboxAddedEvent      ] - Discovery Result with UID 'fmiweather:observation:station_100997_Kirkkonummi_Makiluoto' has been added.
23:56:11.518 [INFO ] [ig.discovery.internal.PersistentInbox] - Added new thing 'fmiweather:observation:station_100997_Kirkkonummi_Makiluoto' to inbox.
23:56:32.782 [ERROR] [enhab.core.thing.binding.ThingFactory] - Thing factory (class org.openhab.binding.fmiweather.internal.HandlerFactory) returned null on create thing when it reports to support the thing type (fmiweather:forecast).
23:56:32.782 [WARN ] [ig.discovery.internal.PersistentInbox] - Cannot create thing. No binding found that supports creating a thing of type fmiweather:forecast.
```

In addition, it is not possible to create the things using MainUI

![image](https://user-images.githubusercontent.com/315866/98434082-23762400-20d5-11eb-99c0-7329fa8468fe.png)




## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1. Settings -> Things -> Inbox -> select any FMI weather thing -> Add as thing. OK to accept suggested thing name
2. Error appears

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->
<!-- Providing context helps us come up with a solution that is most useful in the real world -->

FMI weather unusable in openHAB3

Tried with fresh manual install of openHAB 2.5.10, work flawlessly there so it appears to be regression in OH3.

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):

Fedora linux

```
openjdk version "11.0.9" 2020-10-20
OpenJDK Runtime Environment 18.9 (build 11.0.9+11)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.9+11, mixed mode, sharing)
```

openHAB 3.0 M2, manual runtime installation, with `openhab-addons-3.0.0.M2.kar` in `addons`. Only FMI weather binding installed

@cweitkamp oddly I get the same error with both thing types, `observation` and `forecast`:
```
09:52:39.648 [ERROR] [enhab.core.thing.binding.ThingFactory] - Thing factory (class org.openhab.binding.fmiweather.internal.HandlerFactory) returned null on create thing when it reports to support the thing type (fmiweather:forecast).
09:52:39.648 [WARN ] [ig.discovery.internal.PersistentInbox] - Cannot create thing. No binding found that supports creating a thing of type fmiweather:forecast.
09:52:45.209 [ERROR] [enhab.core.thing.binding.ThingFactory] - Thing factory (class org.openhab.binding.fmiweather.internal.HandlerFactory) returned null on create thing when it reports to support the thing type (fmiweather:observation).
09:52:45.210 [WARN ] [ig.discovery.internal.PersistentInbox] - Cannot create thing. No binding found that supports creating a thing of type fmiweather:observation.
```

--- a/bundles/org.openhab.binding.fmiweather/src/main/resources/OH-INF/thing/thing-types.xml
+++ b/bundles/org.openhab.binding.fmiweather/src/main/resources/OH-INF/thing/thing-types.xml
@@ -279,7 +279,7 @@
 			<channel id="weather-id" typeId="weather-id-channel"/>
 		</channels>
 	</channel-group-type>
-	<channel-group-type id="group-forecast-advanced" advanced="true">
+	<channel-group-type id="group-forecast-advanced">
 		<label>Forecast</label>
 		<description>This is hourly weather forecast.</description>
 		<channels>

